### ISSUE TITLE
Cannot build in low-memory mode

### ISSUE THREAD

**Issue by fredfortier** [tdlib/td#864]
I generated this script to build on Debian 10 in a Docker container:
```Dockerfile
FROM rust:1.40-buster
RUN apt-get update && \
    apt-get install -y curl make git zlib1g-dev libssl-dev gperf php cmake clang libc++-dev libc++abi-dev
```
```sh
git clone https://github.com/tdlib/td.git
cd td
rm -rf build
mkdir build
cd build
export CXXFLAGS="-stdlib=libc++"
CC=/usr/bin/clang CXX=/usr/bin/clang++ cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX:PATH=../tdlib ..
cmake --build . --target prepare_cross_compiling
cd ..
php SplitSource.php
cd build
cmake --build . --target install
cd ..
php SplitSource.php --undo
cd ..
pwd
ls -lht ./td/tdlib/lib
```
But this error occurs:
```
[ 74%] Building CXX object CMakeFiles/tdjson_private.dir/td/generate/auto/td/telegram/td_api_json3.cpp.o
/root/td/td/generate/auto/td/telegram/td_api_json3.cpp:934:75: error: use of undeclared identifier 'lazy_to_json'
  downcast_call(const_cast<Object &>(object), [&jv](const auto &object) { lazy_to_json(jv, object); });
```
The script generated without the low-memory option works. All other things are the same.

**Comment by levlam** [comment]
This will be fixed before 1.6.0 release. Use stable TDLib 1.5.0 for now:
```
git checkout v1.5.0
```

**Comment by levlam** [comment]
master branch is also fixed now.
